src/ide/index.html

```diff
@@ -5,7 +5,7 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		<title>TScript IDE</title>
 	</head>
-	<body>
+	<body style="overflow: hidden">
```

Comment:
What is this for? Shouldn't it rather be in `ide.css`?

Reply:
This might also be a little unrelated, it removes the double scrollbars in the documentation, those only appear in Firefox. I think that it is a bug/rounding error in the CSS engine of Firefox. It might be suitable for another PR, what do you think?

Reply:
If it's unrelated, I think it should be separated. But if nobody else complains just keep it here and do it differently in the future 🤷🏼‍♂️